service/ArbitrageService.js

```diff
@@ -88,16 +109,22 @@ class ArbitrageService {
 			}
 			// swap from usd to blxm
 			await this._ethContracts.poolContract.swapStablesToToken(usdcSwapAmount);
+			logger.info("Swap " + ethers.utils.formatEther(usdcSwapAmount) + " USDC to BLXM in the Binance Smart Chain Network to perform arbitrage trade");
 
 			arbitrageBlxmBalance = await this._ethContracts.blxmTokenContract.getTokenBalance(constants.ARBITRAGE_WALLET_ADDRESS);
 
+			logger.info("Add BLXM to the liquidity pool to complete arbitrage cycle");
 			return await this._bridgeAndSwapBSC(amount, arbitrageBlxmBalance);
 		}
 	}
 
 	async startArbitrageTransferFromETHToBSC(amount, arbitrageBlxmBalance, poolPriceBsc, poolPriceEth, balanceUsdc) {
 		// is liqudity avaible ? 
 		if (!arbitrageBlxmBalance.isZero()) {
+
+			logger.warn("No liqudity of BLXM in the Ethereum network available" + '\n' +
```

Comment:
in der if Abfrage ist ein logischer not operator, das heißt wir haben an dieser Stelle Liquidität
```javascript
 		if (!arbitrageBlxmBalance.isZero()) {
```